src/install/lifecycle_script_runner.zig

```diff
@@ -145,15 +151,15 @@ pub const LifecycleScriptSubprocess = struct {
         }
         const spawn_options = bun.spawn.SpawnOptions{
             .stdin = .ignore,
-            .stdout = if (this.manager.options.log_level.isVerbose())
+            .stdout = if (this.manager.options.log_level.isVerbose() or (this.log_level != .silent and this.foreground))
```

Comment:
log_level == .silent should be `.ignore`

Reply:
updated